regions/shapes/point.py

```diff
@@ -69,16 +68,66 @@ class PointSkyRegion(SkyRegion):
     """
 
     def __init__(self, center, meta=None, visual=None):
-        # TODO: test that center is a 0D SkyCoord
-        self.center = center
+        if center.isscalar:
+            self.center = center
+        else:
+            raise ValueError('the centre should be a 0D SkyCoord object')
```

Comment:
should capitalize The, and `centre` should be `center` (for consistency with the code)